blip-tap.md

```diff
@@ -924,144 +924,211 @@ accept if favorable
+* `out_asset_to_btc_scale` is the scale that defines the fractional component of
+  the out-asset to BTC conversion rate.
 
 ##### Requirements
 
 The sender:
 
-  - MUST set `rfq_id` to the matching `rfq_id` sent in a prior `tap_rfq` message
+- MUST set `rfq_id` to the matching `rfq_id` sent in a prior `tap_rfq` message.
 
-  - MUST set `accepted_rate_tick` to a value they deem to be an acceptable
-    exchange rate
+- MUST set `expiry` to a future unix timestamp after which the quote is no
+  longer valid.
 
-  - MUST set `expiry` to a future unix timestamp after which the quote is no
-    longer valid
+- MUST set `rfq_sig` to be a BIP-340 schnorr signature over the serialized
+  contents of the message without the `rfq_sig` serialized, using their node
+  public key.
 
-  - MUST set `rfq_sig` to be a BIP-340 schnorr signature over the serialized
-    contents of the message without the `rfq_sig` serialized, using their node
-    public key
+- MUST set the fields `in_asset_to_btc_value`, `in_asset_to_btc_scale`,
+  `out_asset_to_btc_value`, and `out_asset_to_btc_scale` to appropriate values
+  based on the asset exchange rates.
 
 The recipient:
 
-  - MUST abandon the attempt if `rfq_sig` is invalid
+- MUST abandon the RFQ session if `version` corresponds to an unsupported
+  version.
 
-  - MUST abandon the attempt if they deem that `accepted_rate_tick` is
-    unreasonable
+- MUST abandon the RFQ session if `rfq_sig` is invalid.
 
-  - SHOULD no longer attempt to utilize the cleared quote after unix timestamp
-    `expiry`
+- MUST abandon the attempt if they deem that `in_asset_to_btc_value`,
+  `in_asset_to_btc_scale`, `out_asset_to_btc_value`, or `out_asset_to_btc_scale`
+  is unreasonable.
+
+- SHOULD no longer attempt to utilize the cleared quote after unix timestamp
+  `expiry`.
 
 #### Rejecting Quotes  (`tap_rfq_reject`) 
 
 In the event that an edge node is unable to satisfy a quote request, then they
 should send `tap_rfq_reject`, identifying the rejected quote ID. A quote might
 be rejected if the channel cannot accommodate the proposed volume, or if the
-edge node is unwilling to carry any HTLCs for that `asset_id`.
+edge node is unwilling to carry any HTLCs for the subject TAP asset(s).
 
-1. type: ?? (`tap_req_accept`)
+1. type: ?? (`tap_rfq_reject`)
 2. data:
+     * [`1*byte`:`version`]
      * [`32*byte`:`rfq_id`]
 
 where:
 
-* `rfq_id` is the quote ID that they wish to reject
+* `version` is the version of the quote reject message. The current version is
+  `0`.
+* `rfq_id` is the quote ID that they wish to reject.
 
 ##### Requirements
 
 The sender:
 
-  - MUST set `rfq_id` to the matching `rfq_id` sent in a prior `tap_rfq` message
+- MUST set `rfq_id` to the matching `rfq_id` sent in a prior `tap_rfq`
+  message.
 
 The recipient:
 
-  - MUST not attempt to send/receive using the rejected `rfq_id`
-
+- MUST not attempt to send/receive using the rejected `rfq_id`.
 
 #### First Hop TAP HTLC Onion Processing
```

Comment:
I think we should rename this to something that reflects: edge node receives tap asset and forwards btc into the lightning network.